Mjml.Net/Components/Body/ButtonComponent.cs

```diff
@@ -0,0 +1,170 @@
+                ["align"] = AttributeTypes.String,
+                ["color"] = AttributeTypes.Color,
+                ["container-background-color"] = AttributeTypes.Color,
+                ["background-color"] = AttributeTypes.Color,
+                ["padding"] = AttributeTypes.PixelsOrPercent,
+                ["padding-bottom"] = AttributeTypes.PixelsOrPercent,
+                ["padding-left"] = AttributeTypes.PixelsOrPercent,
+                ["padding-right"] = AttributeTypes.PixelsOrPercent,
+                ["padding-top"] = AttributeTypes.PixelsOrPercent,
+                ["border-top"] = AttributeTypes.Pixels,
+                ["border-right"] = AttributeTypes.Pixels,
+                ["border-bottom"] = AttributeTypes.Pixels,
+                ["border-left"] = AttributeTypes.Pixels,
+                ["border"] = AttributeTypes.String,
+                ["border-radius"] = AttributeTypes.Pixels,
+                ["inner-padding"] = AttributeTypes.String,
+                ["font-family"] = AttributeTypes.String,
+                ["font-size"] = AttributeTypes.Pixels,
+                ["font-style"] = AttributeTypes.String,
+                ["font-weight"] = AttributeTypes.String,
+                ["letter-spacing"] = AttributeTypes.Pixels,
+                ["line-height"] = AttributeTypes.PixelsOrPercent,
+            };
+
+        public override Attributes? DefaultAttributes { get; } =
+            new Attributes
+            {
+                ["target"] = "_blank",
+                ["text-align"] = "center",
+                ["text-decoration"] = "none",
+                ["text-transform"] = "none",
+                ["vertical-align"] = "middle",
+                ["align"] = "center",
+                ["color"] = "#FFFFFF",
+                ["background-color"] = "#414141",
+                ["padding"] = "10px 25px",
+                ["border"] = "none",
+                ["border-radius"] = "3px",
+                ["inner-padding"] = "10px 25px",
+                ["font-family"] = "Ubuntu, Helvetica, Arial, sans-serif",
+                ["font-size"] = "13px",
+                ["font-weight"] = "normal",
+                ["line-height"] = "120%",
+            };
+
+        public override void Render(IHtmlRenderer renderer, INode node)
+        {
+            string? backgroundColor = node.GetAttribute("background-color");
+            string? width = node.GetAttribute("width");
+            string? height = node.GetAttribute("height");
+            string? href = node.GetAttribute("href");
+            string? fontStyle = node.GetAttribute("font-style");
+            string? borderRadius = node.GetAttribute("border-radius");
+            string? textAlign = node.GetAttribute("text-align");
+            string? innerPadding = node.GetAttribute("inner-padding");
+
+            string buttonHtmlTag = !string.IsNullOrEmpty(href) ? "a" : "p";
+
+            renderer.ElementStart("table")
+                .Attr("border", "0")
```

Comment:
I have put style first, then attr, but I am fine with both. I can also change my code,  we should make it just consistent.

Reply:
Agreed, I only went with attributes first since it's usually the order of writing html `tag -> attributes -> style`. I'm happy to change on my end.

Reply:
As you wish, I can also change my code. I think your order is better ;)